life/conway/conway.go

```diff
@@ -116,10 +116,10 @@ func Draw(ap *ansipixels.AnsiPixels, c *Conway) {
 			switch {
 			case p1 == p2:
 				ap.WriteRune(ansipixels.FullPixel)
-			case p1 == 1:
-				ap.WriteRune(ansipixels.TopHalfPixel)
-			default:
+			case p2 == 1:
```

Comment:
swapped the 2 case so I didn't have to put `switch p1 { case p2:` which I think is less clear than p1==p2